.github/workflows/ci.yaml

```diff
@@ -39,3 +39,5 @@ jobs:
         run: yarn test:ts
       - name: Lint Prettier
         run: yarn lint:prettier
+      - name: Lint ESLint
+        run: yarn lint
```

Comment:
<3

Reply:
Don't worry I plan to switch to pnpm :)))